### DISCUSSION TITLE
"Server code" box

### DISCUSSION THREAD

**Discussion by rejetto** [rejetto/hfs#332] [Show and tell]
Thanks to the new "server code" section in the Options page (arriving with version 0.48), you'll be able to make small customizations very quickly.

<img width="695" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/4de6be30-927c-4c29-a335-b75bdb4a6321">

**Comment by PH5HP** [comment]
Hi,

Can you explain this a little bit in more detail?

Kind regards,
Henk - ftp.ph5hp.nl

**Comment by rejetto** [comment]
it allows you to customize the behavior of the server at a very low level.
it's a feature mostly for programmers, but nonetheless one can get a script by getting help here on the forum, and just copy&paste.
all the capabilities are described at https://github.com/rejetto/hfs/blob/main/dev-plugins.md
for example, with some work you could get SMS notification on uploads (by calling an external SMS service).
